lang/en/local_extension.php

```diff
@@ -225,6 +225,8 @@
 $string['pluginname'] = 'Activity extensions';
 $string['preference_mail_digest'] = 'Mail digest';
 $string['preference_mail_digest_help'] = 'If enabled, instead of sending e-mail notifications immediately they will be grouped into one single message and sent daily.';
+$string['preference_export_csv'] = 'Export CSV';
```

Comment:
Minor comment: I usually organise the "string" alphabetically by the key but that's personal choice.